package test.revolut.exceptions

class UserNotFoundException(val uuid:String) extends GeneralException{
  
  
  
}